app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -111,6 +124,17 @@ def process_server_entity(server, entity)
         end
       end
 
+      def process_availability(availability)
+        case
+        when availability.nil?, availability['value'] == 'unknown'
+          _('Unknown')
```

Comment:
iiic, you are using here translation/i18n, but this is a backend parser. 
translation typically belongs to frontend to allow flexibility per several different languages for different users.
if the string is translated here once in backend, that ability would be lost.